PyTorch/SpeechSynthesis/FastPitch/fastpitch/data_function.py

```diff
@@ -386,14 +404,21 @@ def __call__(self, batch):
 
         audiopaths = [batch[i][7] for i in ids_sorted_decreasing]
 
+        if batch[0][8] is not None:
```

Comment:
I imagine this is the bit that would need updating once the other code is merged?